sdk/eventhub/azure-eventhub/tests/livetest/asynctests/test_send_async.py

```diff
@@ -214,8 +233,8 @@ async def test_send_non_ascii_async(connstr_receivers):
     # receive_message_batch() returns immediately once it receives any messages before the max_batch_size
     # and timeout reach. Could be 1, 2, or any number between 1 and max_batch_size.
     # So call it twice to ensure the two events are received.
-    partition_0 = [EventData._from_message(x) for x in receivers[0].receive_message_batch(timeout=5000)] + \
-                  [EventData._from_message(x) for x in receivers[0].receive_message_batch(timeout=5000)]
+    partition_0 = [EventData._from_message(x) for x in receivers[0].receive_message_batch(timeout=5)] + \
```

Comment:
wondering why you reduced these + other timeouts?

Reply:
the time unit in python amqp is seconds instead of miliseconds now